pages/special-projects/centralized-calendar/end-user.mdx

```diff
@@ -0,0 +1,75 @@
+import { Callout } from "nextra/components";
+
+# End User Guide
+
+<Callout type="info">
+  This page is intended for members of other ICS organization to provide information on how to
+  participate in the ics-centralized-calendar project.
+</Callout>
+
+{/* Include a breif introduction of what is Centralized Calendar Project is and steps to get a the required Google Calendar URL. */}
+
+## Introduction
+
+The `ics-centralized-calendar` project is an easy to use API meant to provide centralized access to all ICS calendars, offering options for selecting specific clubs and output formats (ICS or JSON).
+
+## Steps to Add Your Google Calendar URL
+
+To participate in the ics-centralized-calendar project, follow these steps to share your Google Calendar URL:
+
+1. **Open Google Calendar:**
+
+   - Go to [Google Calendar](https://calendar.google.com).
+
+2. **Access Calendar Settings:**
+
+   - On the left sidebar, hover over the calendar you want to share.
+   - Click on the three vertical dots that appear and select "Settings and sharing."
+
+3. **Get the Shareable Link:**
+
+   - Scroll down to the "Access permissions for events" section.
+   - Check the box that says "Make available to public."
+   - Scroll further down to the "Integrate calendar" section.
+   - Copy the "Public address in iCal format" link.
+
+4. **Submit Your Calendar Link:**
+   - Send the copied calendar link to our project team leads ([Alex](https://github.com/alexanderl19), [Eddy](https://github.com/ecxyzzy)).
+
+<Callout type="tip">
+  Remember to make sure your calendar is set to public so that we can access and integrate your
+  events.
+</Callout>
+
+{/* Include a breif FAQ with information on how event updates work. */}
+
+## FAQ
+
+### How do event updates work?
+
+Once your calendar link is submitted and integrated into the ics-centralized-calendar, your events will be automatically updated every 6 hours. This ensures that any new events or changes to existing events that have not happened yet are reflected in the ics-centralized-calendar quickly.
```

Comment:
Is specifying the exact time period too specific. I'm worried that this information can become quickly out of date. Also as a separate question, was there a reason we settled on 6 hours? That seems like a very long period to me.

Maybe something like "... your events will be updated automatically. Updates may take up to x hours to reflect on our end." adds more clarity?